integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -226,14 +246,36 @@ def _get_most_recent_catalog_update_time(self):
             return past_transmissions[0]['content_last_changed']
         return None
 
+    def update_content_transmissions_catalog_uuids(self):
+        """
+        Placeholder
+        """
+        enterprise_customer_catalogs = self.enterprise_configuration.customer_catalogs_to_transmit or \
+            self.enterprise_customer.enterprise_customer_catalogs.all()
+
+        for enterprise_customer_catalog in enterprise_customer_catalogs:
+            content_metadata_items = self.enterprise_catalog_api.get_content_metadata(
+                self.enterprise_customer,
+                [enterprise_customer_catalog]
+            )
+            content_ids = [get_content_metadata_item_id(item) for item in content_metadata_items]
+            ContentMetadataItemTransmission = apps.get_model(
+                'integrated_channel',
+                'ContentMetadataItemTransmission'
+            )
+            transmission_items = ContentMetadataItemTransmission.objects.filter(enterprise_customer=self.enterprise_configuration.enterprise_customer, integrated_channel_code=self.enterprise_configuration.channel_code(), content_id__in=content_ids)
+            for item in transmission_items:
+                item.enterprise_customer_catalog_uuid = enterprise_customer_catalog.uuid
+                item.save()
 
 class ContentMetadataItemExport:
     """
     Object representation of a content metadata item export.
     """
 
-    def __init__(self, content_metadata_item, channel_content_metadata_item, content_last_changed=None):
+    def __init__(self, content_metadata_item, channel_content_metadata_item, enterprise_customer_catalog_uuid=None, content_last_changed=None):
         self.content_id = get_content_metadata_item_id(content_metadata_item)
         self.metadata = content_metadata_item
         self.channel_metadata = channel_content_metadata_item
         self.content_last_changed = content_last_changed
+        self.enterprise_customer_catalog_uuid = enterprise_customer_catalog_uuid
```

Comment:
Is it important to undestand when/why `enterprise_customer_catalog_uuid` would be None?  Might want to log instances of that particular scenario.

Reply:
should None raise a failure / exception?

Reply:
yeah, I agree- we should _**always**_ have a catalog uuid here. I'll make it a required param